Remotion/Mixins/Core/CodeGeneration/TypePipe/ExpressionMixinContextOriginSerializer.cs

```diff
@@ -43,7 +43,7 @@ public Expression CreateNewExpression ()
       return Expression.New (
           s_constructor,
           Expression.Constant (Kind),
-          Expression.Call (s_assemblyLoadMethod, Expression.Constant (Assembly.FullName)),
+          Expression.Call (s_assemblyLoadMethod, Expression.Constant (Assembly!.FullName)),
```

Comment:
Didn't we intend to add the null-guard in Assembly-getter?

Reply:
We did, just not yet. I did add [RM-7691](https://re-motion.atlassian.net/browse/RM-7691).

Reply:
Okay, would be great if you could add this ticket as a todo to the code.

Reply:
Where should I add the TODOs? Above all occurrences where `!` needs to be used?

Reply:
above or inline, depending on the statement length etc. probably mostly above.

Reply:
Oh, before I forget: We only need the TODO at the usage site if the compiler ins't going to complain about this anyway once we make the Assembly-property not-nullable. The TODO would then only be needed with the property declaration, not the usages. (and I hope I'm consistent aobut this...)

Reply:
That's what I did in https://github.com/re-motion/Framework/commit/f6be052fc0baea0369517e17ce5b8c2c3e95d5cc if I understood the last comment alright.

Reply:
Ah okay, so it's inline now at the usage site. okay. Hmm...looks like I didn't see the "outdated"  at the top of the file here.